code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -477,4 +477,4 @@ emp_act
 /mob/living/carbon/human/water_act(volume, temperature, source)
 	..()
 	if(temperature >= 330)	bodytemperature = bodytemperature + (temperature - bodytemperature)
-	if(temperature <= 280)	bodytemperature = bodytemperature - (bodytemperature - temperature)
\ No newline at end of file
+	if(temperature <= 280)	bodytemperature = bodytemperature - (bodytemperature - temperature)

```

Comment:
EOF got moved down, possibly automatically by whatever git spacemanspark is using.